ui/admin/app/templates/scopes/scope/roles/role/principals.hbs

```diff
@@ -46,7 +45,7 @@
           {{#each @model.principals as |principal|}}
             <body.row as |row|>
               <row.headerCell>
-                {{principal.displayName}}
+                <LinkToPrincipal @model={{principal}} />
```

Comment:
We should probably have an ability check here to ensure the user has read permission, otherwise show the name unlinked.

Reply:
Good point. will add the ability!